post-templates/embertimes-template.md

```diff
@@ -2,18 +2,17 @@
 title: The Ember Times - Issue No. XXS
 author:
   - the-crowd # replace with real authors from the author folder (add yourself if you're not there)
+date: 2021-XX-XXT00:00:00.000Z
 tags:
-  - ewsletter
-  - ember-js-times
+  - newsletter
+  - ember-times
```

Comment:
It seems like we dno't really use `ember-times` but if we do, remove the `js`?

Reply:
Maybe skip it based on the issue 170?
```suggestion
```